src/main/java/org/nguh/nguhcraft/block/NguhBlocks.kt

```diff
@@ -235,6 +227,293 @@ object NguhBlocks {
             .mapColor(MapColor.STONE_GRAY)
     )
 
+    // Cinnabar blocks
+    val CINNABAR = Register(
+        "cinnabar",
+        ::Block,
+        AbstractBlock.Settings.copy(Blocks.TUFF)
+            .mapColor(MapColor.DARK_RED)
+    )
+
+    val CINNABAR_SLAB = RegisterVariant(CINNABAR, "slab", ::SlabBlock)
+    val CINNABAR_STAIRS = RegisterStairs(CINNABAR)
```

Comment:
Please refactor all other stairs, slabs, and walls to be declared with `RegisterVariant()` (or for stairs, `RegisterStairs()`).